### ISSUE TITLE
[3.31.0] CI builds fails, command not found: bob

### ISSUE THREAD

**Issue by efstathiosntonas** [software-mansion/react-native-screens#2108]
### Description

After running `yarn install` on CI on a  macos 14 machine the install fails with: 

`react-native-screens@npm:3.31.0 [e76ba] couldn't be built successfully (exit code 127, logs can be found here: /private/var/folders/cm/2r_39vcd593436b3_ht4nyqw0000gp/T/xfs-1e614b9e/build.log)`

the log states: 

`command not found: bob`

### Steps to reproduce

1. I'm using yarn 4.1.1
2. use latest version of react-native-screens 3.31.0
3. yarn install fails

### Snack or a link to a repository

no-repo-needed

### Screens version

3.31.0

### React Native version

0.73.7

### Platforms

Android, iOS

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Paper (Old Architecture)

### Build type

Release mode

### Device

Real device

### Acknowledgements

Yes

**Comment by keuvy** [comment]
just happened to me too

`node_modules/react-native-screens: Command failed.
Exit code: 127
Command: bob build && husky install`

`/bin/sh: bob: command not found`

**Comment by copiiinux** [comment]
Experiencing the same behaviour on Windows and RN@0.74
```
npm ERR! code 1
npm ERR! path C:\Users\XXXX\projects\lahoco-mobile\node_modules\react-native-screens
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c bob build && husky install
npm ERR! 'bob' is not recognized as an internal or external command,
npm ERR! operable program or batch file.
[...]
Process finished with exit code 1
```

**Comment by mdenizkaba** [comment]
Command failed.
Exit code: 127
Command: bob build && husky install
Arguments:
Directory: /Users/muhammetkaba/kaba/tiktok/node_modules/react-native-screens
Output:
/bin/sh: bob: command not found

**Comment by FantWings** [comment]
**same here** react native 0.74.0
error /Users/zhijun/Documents/github/furries-client-rn/node_modules/react-native-screens: Command failed.
Exit code: 127
Command: bob build && husky install
Arguments: 
Directory: /Users/zhijun/Documents/github/furries-client-rn/node_modules/react-native-screens

**Comment by chinomnsoawazie** [comment]
> Command failed. Exit code: 127 Command: bob build && husky install Arguments: Directory: /Users/******/node_modules/react-native-screens Output: /bin/sh: bob: command not found

Same! Happened after I upgraded to rn `0.74.0`

**Comment by tarunsmartdata** [comment]
error /node_modules/react-native-screens: Command failed.
Exit code: 127
Command: bob build && husky install
Arguments: 
Directory: /node_modules/react-native-screens
Output:

**Comment by efstathiosntonas** [comment]
can someone install `react-native-builder-bob` globally and try to install rn-screens afterwards?

**Comment by Acanguven** [comment]
@efstathiosntonas 

```
npm ERR! command sh -c bob build && husky install
npm ERR! ℹ Building target commonjs
npm ERR! ℹ Cleaning up previous build at lib/commonjs
npm ERR! ℹ Compiling 61 files in src with babel
npm ERR! ✔ Wrote files to lib/commonjs
npm ERR! ℹ Building target module
npm ERR! ℹ Cleaning up previous build at lib/module
npm ERR! ℹ Compiling 61 files in src with babel
npm ERR! ✔ Wrote files to lib/module
npm ERR! ℹ Building target typescript
npm ERR! ℹ Cleaning up previous build at lib/typescript
npm ERR! ℹ Generating type definitions with tsc
npm ERR! ✖ Couldn't find a tsconfig.json in the project root.
npm ERR! bob build
```

**Comment by mehmetdemiray** [comment]
@efstathiosntonas 

Output:
```
ℹ Building target commonjs
ℹ Cleaning up previous build at lib/commonjs
ℹ Compiling 61 files in src with babel
✔ Wrote files to lib/commonjs
ℹ Building target module
ℹ Cleaning up previous build at lib/module
ℹ Compiling 61 files in src with babel
✔ Wrote files to lib/module
ℹ Building target typescript
ℹ Cleaning up previous build at lib/typescript
ℹ Generating type definitions with tsc
✖ Couldn't find a tsconfig.json in the project root.
bob build
```

**Comment by mehmetdemiray** [comment]
I solved problem with changing version for now. I think this version is working when build

3.31.0-rc.1

`npm i react-native-screens@3.31.0-rc.1`
or
`yarn add react-native-screens@3.31.0-rc.1`

**Comment by zerobertelprivat** [comment]
Hello,

same problem here with version 3.31.0 (using RN 0.74.0 / yarn 1.22.22).
Cannot do "yarn add react-native-screens" ....
The problem doesnt exit with version 3.31.0-rc.1

Thank you,
Robert

**Comment by audunjl** [comment]
Hello,
Same here. 
I'm able to install react-native-screens@3.31.0-rc.1
However, when trying to install @react-navigation/native-stack the same error occurs, even though I've already installed react-native-screens 3.31.0-rc.1 successfully.

**Comment by tboba** [comment]
Hello everyone! We're currently aware of this issue with @kkafar. It was mistake on our side, since we've bumped Yarn to v4 version and didn't realize that it triggers `postinstall` step 😄 
Sorry, we'll be releasing a patch version in upcoming hour.

**Comment by Henus321** [comment]
npm ERR! code 127
npm ERR! path node_modules/react-native-screens
npm ERR! command failed
npm ERR! command sh -c bob build && husky install
npm ERR! sh: bob: command not found

**Comment by kkafar** [comment]
@tboba Released 3.31.1. Let us know, whether the issue disappeared

**Comment by gremech** [comment]
The problem is solved in 3.31.1
